packages/table/src/interactions/reorderable.tsx

```diff
@@ -112,22 +112,27 @@ export class DragReorderable extends React.Component<IDragReorderable, {}> {
         const { selectedRegions } = this.props;
 
         const selectedRegionIndex = Regions.findContainingRegion(selectedRegions, region);
-        if (selectedRegionIndex < 0) {
-            return false;
-        }
-
-        const selectedRegion = selectedRegions[selectedRegionIndex];
-        if (Regions.getRegionCardinality(selectedRegion) !== cardinality) {
-            // ignore FULL_TABLE selections
-            return false;
+        if (selectedRegionIndex >= 0) {
+            const selectedRegion = selectedRegions[selectedRegionIndex];
+            if (Regions.getRegionCardinality(selectedRegion) !== cardinality) {
+                // ignore FULL_TABLE selections
+                return false;
+            }
+
+            // cache for easy access later in the lifecycle
+            const selectedInterval = isRowHeader ? selectedRegion.rows : selectedRegion.cols;
+            this.selectedRegionStartIndex = selectedInterval[0];
+            // add 1 to correct for the fencepost
```

Comment:
Nitpick -- can you be a little bit more descriptive here? A "because selected interval is inclusive" or "includes its endpoints" or something would be ok; just looking for a why-the-fencepost.